packages/modelviewer.dev/data/examples.json

```diff
@@ -13,15 +13,15 @@
       },
       {
         "htmlId": "demo-container-3",
-        "name": "Customize Loading"
+        "name": "Customize Load"
```

Comment:
I'm just going to put these comments at arbitrary code locations, since they're actually design feedback. If I leave them as top-level comments, they're hard to respond to.

Let's swap Documentation and Examples; I think we'll land on documentation first, so it makes sense to be on the left.

Reply:
done